src/pip/_internal/req/req_install.py

```diff
@@ -802,6 +802,7 @@ def prepend_root(path):
                     new_lines.append(
                         os.path.relpath(prepend_root(filename), egg_info_dir)
                     )
+            new_lines = sorted(new_lines)
```

Comment:
Better to make this an actual list (`new_lines.sort()`) rather than a generator? I guess the generator works at the moment, but it may cause confusion for future maintenance that `new_lines` isn't an actual list after this point.

Reply:
`sorted` returns a list https://docs.python.org/3.6/library/functions.html#sorted but indeed calling `new_lins.sort()` should be enough since it is already a list.

Reply:
Ouch. Sorry, I was getting confused between `sorted` and `reversed`... Thanks for the correction.

Reply:
Sure, doesn't make a difference to me... Latest version of this branch uses `new_lines.sort()`.